chapters/annotations.tex

```diff
@@ -109,10 +109,11 @@ \subsection{Annotations for Figures}\label{annotations-for-figures}
 The \lstinline!identifier! in \lstinline!Figure! and \lstinline!Plot! is an optional \lstinline!String! identifier, and is intended to identify the
 \lstinline!Figure! and \lstinline!Plot! for programmatic access.  The \lstinline!figures! annotation is inherited in the sense that each class has
 a collection of figures comprised by the contents of the \lstinline!figures! annotation in the class itself, as well as the \lstinline!figures! annotations
-from any base classes.  A \lstinline!Figure! must be uniquely identified by its \lstinline!identifier! and a class having it in its collection.  This
+from any base classes. A \lstinline!Figure! must be uniquely identified by its \lstinline!identifier! and a class having it in its collection.  This
 means that a \lstinline!Figure! \lstinline!identifier! must be unique among all \lstinline!Figure! annotations within the same \lstinline!figures!
 annotation as well as among all \lstinline!figures! annotations from inherited classes.  A \lstinline!Plot! \lstinline!identifier! on the other hand
 is only required to be unique among the \lstinline!plots! in the the same \lstinline!Figure! annotation.
+If the identifier is an empty string it cannot be used for programmatic access and is exempt from the uniqueness requirement.
```

Comment:
In addition to perhaps moving this to before we go into the different details for `Figure` and `Plot` identifiers, the _requirement_ should be in the plural since we have both `Figure` and `Plot` requirements.

Reply:
Ok for second part.
For the first part: I tried having it before that discussion, but it looked odd.

Reply:
This discussion is now obsolete.  Resolving.